gateway/src/apicast/policy/3scale_batcher/keys_helper.lua

```diff
@@ -5,7 +5,7 @@ local concat = table.concat
 local sort = table.sort
 local unpack = table.unpack
 local ngx_re = ngx.re
-local table = table
+local new_tab = require('resty.core.base').new_tab
```

Comment:
@mikz I remember that we discussed this in a previous PR.
I needed to require this in order to make the test pass. I guess it has to do with the OpenResty upgrade that we merged today.

Reply:
👍